packages/superset-ui-chart/src/models/ChartProps.ts

```diff
@@ -58,17 +58,17 @@ const DEFAULT_HEIGHT = 600;
 export default class ChartProps {
   static createSelector: () => ChartPropsSelector;
 
-  annotationData: AnnotationData;
+  annotationData?: AnnotationData;
```

Comment:
This may have effects to consumer app (or deck.gl plugins that I know consume rawFormData) that expect these values to be valid / crash if they don't check. Do you need this change in this PR?

Reply:
Made this change because when I updated the test data with `ChartProps` obtained from the running app, it throws a missing prop error. Just thought the typing should reflect the actual data. If a consumer app expects these to be present for specific chart types, they should probably extend `ChartProps`. Hope this doesn't break any existing apps since they would have locked package versions?